Misc/NEWS.d/next/IDLE/2017-11-07-23-32-19.bpo-31971.ZpBCeF.rst

```diff
@@ -0,0 +1,3 @@
+Fix IDLE test failure.
+
+On one machine, tk sees a mouse cursor hovering over one widget.
```

Comment:
I'm not sure how this NEWS entry is rendered in HTML. I suggest to put both sentences on the same line.